packages/oui-typography/README.md

```diff
@@ -93,6 +93,15 @@ oui-typography is a package which provides tools as constants and mixins for man
 </div>
 ```
 
+```html:preview
+<div>
+  <a href="#" class="oui-link oui-link_external">
```

Comment:
I don't like the idea to create a modifier to resize a specific icon.
It could be any icon in all the icons.

Reply:
In fact I'd like to know if it's correct to include the icon directly in the modifier. Cause, I hadn't be able to see it in ui-kit ?